src/System.Management.Automation/project.json

```diff
@@ -156,6 +156,7 @@
                 "System.Diagnostics.StackTrace": "4.0.1",
                 "System.Diagnostics.TraceSource": "4.0.0",
                 "System.Dynamic.Runtime": "4.0.11",
+                "System.IO.FileSystem": "4.0.2-beta-24412-03",
```

Comment:
I'm not so sure we should depend on a preview build package in powershell. @lzybkr can you please comment?